site/zenodo_rdm/migrator/extract.py

```diff
@@ -175,3 +175,25 @@ def run(self):
     ) as requests
 ) TO STDOUT;
 """
+
+EXTRACT_COMMUNITIES_SQL = """
```

Comment:
Query was missing. 

I added the query to get communities and their featured information, if any.  In case the community is featured, the query returns a new attribute `is_featured`.

Featured columns were aliased to `featured_<original_column_name>` to avoid some collisions with e.g. community `created` or `updated` columns.